Bottom: 46ad9810d35fa1e8e5d0079402323143da9da45b
Top:    3784a9081337bd31ddc7b6b2315291064c44cf51
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 10:21:49 +0800

Refresh of debug.patch

---

diff --git a/modules/mount_autofs.c b/modules/mount_autofs.c
index 8c1e600..c7f8b3d 100644
--- a/modules/mount_autofs.c
+++ b/modules/mount_autofs.c
@@ -184,6 +184,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 		return 1;
 	}
 	argv[0] = info->map;
+	error(LOGOPT_ANY, "info->type %s", info->type);
 
 	if (options) {
 		p = options;
